.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: Ruby
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Ah, we should probably change this to `main` at some point.

Reply:
Agreed.